recipes/console_shortcut/console_shortcut.json

```diff
@@ -0,0 +1,12 @@
+{
+    "menu_name": "Anaconda${PY_VER} ${PLATFORM}",
```

Comment:
```suggestion
    "menu_name": "Miniforge${PY_VER} ${PLATFORM}",
```

Reply:
Ideally, we would like to use the name of the distribution, is this information available somewhere?

Reply:
> Ideally, we would like to use the name of the distribution, is this information available somewhere?

I would suggest to use `Miniforge` for now and when there is a better solution to improve it in the feedstock.